packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -125,13 +132,9 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
       isStickyBottom !== nextState.isStickyBottom ||
       this.props.stickyPosition !== nextProps.stickyPosition ||
       this.props.children !== nextProps.children ||
-      (this._nonStickyContent &&
-        this._nonStickyContent.current &&
-        ((this._placeHolder &&
-          this._placeHolder.current &&
-          this._nonStickyContent.current.offsetHeight !== this._placeHolder.current.offsetHeight) ||
-          (this.stickyContentTop && this._nonStickyContent.current.offsetHeight !== this.stickyContentTop.offsetHeight) ||
-          (this.stickyContentBottom && this._nonStickyContent.current.offsetHeight !== this.stickyContentBottom.offsetHeight)))) as boolean;
+      isHeightOrWidthDifferent(this._nonStickyContent, this._placeHolder) ||
```

Comment:
Do we need this check?  In thery, nonStickyContent and placeHolder should always keep their width/height identical inside of `_getNonStickyPlaceholderHeight`. (e.g: if it's sticky, then it takes on the width/height of the `nonStickyContent`, otherwise, it doesn't return values and just inherits the height/width without setting them in `_getNonStickyPlaceholderHeight`

Reply:
@leddie24 Check [this](https://microsoft-my.sharepoint.com/:f:/p/hatrived/ErYBJRml-mJNoevBTyv7nAQBJHMHWMMIg_2Ig8vLxfV3Og?e=LVMg44)